Cargo.lock

```diff
@@ -316,40 +329,40 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
 name = "libc"
-version = "0.2.55"
+version = "0.2.65"
```

Comment:
https://github.com/rust-lang/libc/compare/0.2.55...0.2.65

Various improvements, it's a very active project!